app/wiring/AppComponents.scala

```diff
@@ -21,13 +24,33 @@ class AppComponents(context: Context, stage: String) extends BuiltInComponentsFr
     GoogleAuthConfig(clientId, clientSecret, redirectUrl, domain, antiForgeryChecker = AntiForgeryChecker.borrowSettingsFromPlay(httpConfiguration))
   }
 
+  private val googleGroupChecker = {
```

Comment:
worth noting we have a try/catch in `AppLoader` which catches and logs any app startup errors which occur in `AppComponents`